### PR TITLE
Fix build errors for gazebo version 8.0

### PR DESCRIPTION
**Pull request by devbharat**
Check reference [here](http://gazebosim.org/api/8.0.0/classgazebo_1_1rendering_1_1Scene.html) and [here](https://osrf-distributions.s3.amazonaws.com/ign-math/api/1.0.0/classignition_1_1math_1_1Vector3.html)

### CODE REVIEW

**Comment by hitimo** [comment]
ok to test

File: rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp [hunk 1]
```diff
@@ -384,12 +388,21 @@ void GazeboBagPlugin::LogWrenches(const common::Time now) {
     wrench_msg.header.frame_id = collision1_name + "--" + collision2_name;
     wrench_msg.header.stamp.sec = now.sec;
     wrench_msg.header.stamp.nsec = now.nsec;
+#if GAZEBO_MAJOR_VERSION >= 7
```
Comment (burrimi) [inline rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp]: when did this change? I think jenkins uses gazebo7 for 16.04 and it does not compile.

Comment (devbharat) [inline rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp]: My bad, gave error for gazebo8, I'll make them > 7

**Comment by devbharat** [comment]
@burrimi I made these changes to fix build errors I got when compiling them as part of PX4. I also verified that they work when compiled through catkin (using ROS kinetic and gazebo8. Some changes come for example from using ignition::math instead of gazebo::math).

**Comment by burrimi** [comment]
lgtm.

**Comment by devbharat** [comment]
@burrimi is this merge still pending some action on my side?

**Comment by burrimi** [comment]
yes, you need to press the merge button ;)

**Comment by devbharat** [comment]
haha...wish I could :)

**Pull request merged**